gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileExporter.java

```diff
@@ -54,6 +54,7 @@
 @Service
 public class FileExporter {
     private static final Logger LOGGER = LoggerFactory.getLogger(FileExporter.class);
+    private static final String ZIP = ".zip";
```

Comment:
Maybe name it "ZIP_EXT" or something like that to make it clear chat this String represents a "*.zip" file extension

Reply:
done

Reply:
done